src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -31,6 +32,14 @@ class PeerActor(nodeInfo: NodeInfo) extends Actor with ActorLogging {
 
   val peerId = self.path.name
 
+  val waitForStatusInterval = 30.seconds
+  val waitForChainCheck = 15.seconds
+
+  //FIXME move this to props
+  lazy val DaoBlockNumber = 1920000
+  lazy val DaoBlockTotalDifficulty = BigInt("39490964433395682584")
```

Comment:
@rtkaczyk I like puting links in comments (specially when grabbing snippets/gists) ... I will add it